Common/Models/Commands/Command.cs

```diff
@@ -4,6 +4,23 @@ namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Models.Co
 {
     public class Command
     {
+        /// <summary>
+        /// Serialziation deserialziation constructor.
+        /// </summary>
+        internal Command()
```

Comment:
actually you might need an empty constructor for deserialization cause this would clear up parameters right? Parameters = new List<Parameter>();